pandas/core/generic.py

```diff
@@ -10283,8 +10283,8 @@ def _doc_parms(cls):
 Returns
 -------
 %(name1)s or %(name2)s (if level specified)\
-%(see_also)s
-%(examples)s\
+%(see_also)s\
```

Comment:
Do we have an expectation that substituted sections are responsible for line breaks? I feel like this has caused a few issues throughout code base curious if you have thoughts

Reply:
@datapythonista I *might* already have fixed this one in https://github.com/pandas-dev/pandas/pull/26780/files, but by adding a extra newline in the "see also" string instead of here. 
My reasoning was that if you do it here, and no see also is introduced, you might get too many empty lines (but didn't try that actually, so not sure this is correct)

Reply:
This is trickier than what it looks, our validation complains if there are two line breaks together, and also a missing line break at the end. And we don't fail the validation, but we get warnings (and incorrect rendering) if the line break is missing before a section title, as the title is not detected.

It took me a while to find a working solution for all the cases, I think the one here is the only possible one, there are many docstrings affected by this.

Reply:
Yeah, I trust you here ;-) I only fixed that single case (of min/max I think), and didn't check the interaction with fixing others. But you might need to revert my change to get it working with master.

Reply:
I think the change you did was correct and is still needed, it just didn't address all the cases.